Resources/doc/commands/generate_bundle.rst

```diff
@@ -3,7 +3,7 @@ Generating a New Bundle Skeleton
 
 .. caution::
 
-    If your application is based on Symfony 3, replace ``php app/console`` by
+    If your application is based on Symfony 3, replace ``php bin/console`` by
     ``php bin/console`` before executing any of the console commands included
```

Comment:
The current sentence is correct. If you use Symfony 3, you should use `bin/console` instead of `app/console` which is the path for Symfony 2.